src/attributes/codegen.md

```diff
@@ -226,52 +226,52 @@ Reference Manual], or elsewhere on [developer.arm.com].
-`fcma`         | `neon`         | FEAT_FCMA --- Floating point complex number support
-`fhm`          | `fp16`         | FEAT_FHM --- Half-precision FP FMLAL instructions
-`flagm`        |                | FEAT_FlagM --- Conditional flag manipulation
-`fp16`         | `neon`         | FEAT_FP16 --- Half-precision FP data processing
-`frintts`      |                | FEAT_FRINTTS --- Floating-point to int helper instructions
-`i8mm`         |                | FEAT_I8MM --- Int8 Matrix Multiplication
-`jsconv`       | `neon`         | FEAT_JSCVT --- JavaScript conversion instruction
-`lse`          |                | FEAT_LSE --- Large System Extension
-`lor`          |                | FEAT_LOR --- Limited Ordering Regions extension
-`mte`          |                | FEAT_MTE & FEAT_MTE2 --- Memory Tagging Extension
-`neon`         |                | FEAT_FP & FEAT_AdvSIMD --- Floating Point and Advanced SIMD extension
-`pan`          |                | FEAT_PAN --- Privileged Access-Never extension
-`paca`         |                | FEAT_PAuth --- Pointer Authentication (address authentication)
-`pacg`         |                | FEAT_PAuth --- Pointer Authentication (generic authentication)
-`pmuv3`        |                | FEAT_PMUv3 --- Performance Monitors extension (v3)
-`rand`         |                | FEAT_RNG --- Random Number Generator
-`ras`          |                | FEAT_RAS & FEAT_RASv1p1 --- Reliability, Availability and Serviceability extension
-`rcpc`         |                | FEAT_LRCPC --- Release consistent Processor Consistent
-`rcpc2`        | `rcpc`         | FEAT_LRCPC2 --- RcPc with immediate offsets
-`rdm`          |                | FEAT_RDM --- Rounding Double Multiply accumulate
-`sb`           |                | FEAT_SB --- Speculation Barrier
-`sha2`         | `neon`         | FEAT_SHA1 & FEAT_SHA256 --- Advanced SIMD SHA instructions
-`sha3`         | `sha2`         | FEAT_SHA512 & FEAT_SHA3 --- Advanced SIMD SHA instructions
-`sm4`          | `neon`         | FEAT_SM3 & FEAT_SM4 --- Advanced SIMD SM3/4 instructions
-`spe`          |                | FEAT_SPE --- Statistical Profiling Extension
-`ssbs`         |                | FEAT_SSBS & FEAT_SSBS2 --- Speculative Store Bypass Safe
-`sve`          | `fp16`         | FEAT_SVE --- Scalable Vector Extension
-`sve2`         | `sve`          | FEAT_SVE2 --- Scalable Vector Extension 2
-`sve2-aes`     | `sve2`, `aes`  | FEAT_SVE_AES --- SVE AES instructions
-`sve2-sm4`     | `sve2`, `sm4`  | FEAT_SVE_SM4 --- SVE SM4 instructions
-`sve2-sha3`    | `sve2`, `sha3` | FEAT_SVE_SHA3 --- SVE SHA3 instructions
-`sve2-bitperm` | `sve2`         | FEAT_SVE_BitPerm --- SVE Bit Permute
-`tme`          |                | FEAT_TME --- Transactional Memory Extension
-`vh`           |                | FEAT_VHE --- Virtualization Host Extensions
+Feature        | Implicity Enables            | Feature Name
+-------        | -----------------            | ------------
```

Comment:
Implicitly is now misspelled.

Also, can you remove all the addition of spaces in the table? It doesn't look like any of them are needed.